src/widgets/hero/ui/hero.tsx

```diff
@@ -5,12 +5,12 @@ export const Hero = () => {
     <main id="main" className="main container">
       <div className="main content">
         <div className="title-container">
-          <h1 className="title-main">The Rolling Scopes</h1>
-
           <div className="subtitle-container">
             <h2 className="subtitle">an international community of developers</h2>
             <h2 className="subtitle">since 2013</h2>
           </div>
+
+          <h1 className="title-main">The Rolling Scopes</h1>
```

Comment:
Please use css modules with classNames lib.
You can find example [here](https://t.me/c/2142251410/1519)

Reply:
Title and subtitle will be replaced by components as part of the widget refactoring. It is no use to fix classes in this task.